src/test/regress/expected/gp_parallel.out

```diff
@@ -1590,6 +1590,81 @@ select * from t1 order by c2 asc limit 3 offset 5;
 
 abort;
 --
+-- Test Parallel Hash Left Anti Semi (Not-In) Join(parallel-oblivious).
+--
+create table t1(c1 int, c2 int) using ao_row distributed by (c1);
+create table t2(c1 int, c2 int) using ao_row distributed by (c1);
+create table t3_null(c1 int, c2 int) using ao_row distributed by (c1);
+set enable_parallel = on;
+set gp_appendonly_insert_files = 2;
+set gp_appendonly_insert_files_tuples_range = 100;
+set max_parallel_workers_per_gather = 2;
```

Comment:
Next time , here can use alter table to set parallel_workers instead for convenience.

Reply:
I once tried to set it when creating ao table, but it's not supported.  An issue of that is a perfect `good-first-issue`, I will create that.